Explorer RPP Commands

RenzoDD opened the pull request.

Commit:
-added ignore ipfs getassetdata and initial count

Commit:
-added listassetissuances

mctrivia commented:
Thanks for the pull request.  It is a great idea but the code as is will cause problems other places in the code.  Can you make one of the following changes

Option 1:
 Remove the initial parameters from DigiAsset and add the initial in to the json as part of RPC after the toJSON call

Options 2:
 Remove setter for initial, and instead if getter is called and it hasn't been gotten yet make database call.  Also move after to after the is simplified check in to JSON

Commit:
-updated listassetissuances include asset index

mctrivia committed:
fix getassetdata so backwards compatible

Commit:
-added listlastblocks

Commit:
-updated listassets to reverse the list

Commit:
-added total amount on getassetdata

Commit:
-updated moved blockbasics

mctrivia committed:
-fix some bugs
-added tests
-added start to listlastblocks

PLease note: list assets test is not passing

mctrivia commented:
pushed an update to pr-10 that fixes some bugs and adds start parameter to listlastblocks.  We still need tests for listassetissuances before we can merge and Database::getAssetIDsOrderedByIssuanceHeight is throwing an error during tests after your changes to it.  Have not had a chance to look in to why.

Commit:
-added listlastassets -removed listassets

mctrivia committed:
-bug fixes
-added listlastassetspageindexes

mctrivia commented:
Has been manually merged do to conflicts
